recipes/libcufile/meta.yaml

```diff
@@ -20,18 +20,41 @@ source:
 
 build:
   number: 0
-  skip: true  # [not linux]
+  skip: true  # [not (linux64 or aarch64)]
+
+test:
+  commands:
+    # Note that the version of libcufile does not match
+    # {{ version.split(".")[0] }} but the major version of libcufile_rdma does
+    # match that. Also we must drop the last component of the version when
+    # finding the versioned .so file.
+    {% set full_version = version.split(".")[:-1] | join(".") %}
+    - test -f $PREFIX/etc/cufile.json
+    - test -L $PREFIX/lib/libcufile.so.0
+    - test -L $PREFIX/lib/libcufile_rdma.so.{{ version.split(".")[0] }}  # [linux64]
+    - test -L $PREFIX/lib/libcufile.so.{{ full_version }}
+    - test -L $PREFIX/lib/libcufile_rdma.so.{{ full_version }}  # [linux64]
+    - test -L $PREFIX/targets/{{ target_name }}/lib/libcufile.so.0
+    - test -L $PREFIX/targets/{{ target_name }}/lib/libcufile_rdma.so.{{ version.split(".")[0] }}  # [linux64]
+    - test -f $PREFIX/targets/{{ target_name }}/lib/libcufile.so.{{ full_version }}
+    - test -f $PREFIX/targets/{{ target_name }}/lib/libcufile_rdma.so.{{ full_version }}  # [linux64]
 
 outputs:
   - name: libcufile
     build:
       missing_dso_whitelist:
-        - "*/libcuda.so.*"                     # [linux]
-      run_exports:
-        - {{ pin_subpackage("libcufile", max_pin="x") }}
+        - "*/libcuda.so.*"
+        # TODO: The following libraries are required for libcufile_rdma.so,
+        # which is only for linux64 (not aarch64). What provides these?
+        # rdma-core-devel-cos7-x86_64 is close, but doesn't seem to work.
+        # Perhaps because the files are in a sysroot location?
```

Comment:
Hopefully the comments below (mostly) address this

If so, we can simplify this comment to explain the CDT (a package that the system will need to provide and Conda won't)